### PR TITLE
acs: command improvements with new defaults, scale-up, new command group names

### LINKED ISSUE #1090: ACS: move commands from 'az vm container *' to 'az acs'

### LINKED ISSUE #1092: ACS: new default values when create a new ACS
1. private agent count and master count to 3 
2. `--agent-vm-size` should be STANDARD_D2V2

### LINKED ISSUE #1094: ACS: support scale out by updating private agents number
We should at least expose this parameter in the generic update command

### PR DESCRIPTION
**Pull request by yugangw-msft**
Fix #1090, fix #1092, fix #1094

### CODE REVIEW

**Comment by derekbekoe** [comment]
Would the acs commands not need to be moved to an azure-cli-acs command package instead of being in azure-cli-vm?

**Comment by yugangw-msft** [comment]
@derekbekoe, so far there is no need from command capacity's point of view, but if there are needs, we can.  I prefer to let it stay, just from code maintenance's aspect, that VM, VMSS, and ACS still share some common functionalities like SSH key-gen I am getting it work. But again let me know.

**Comment by derekbekoe** [comment]
The way modules and commands a loaded right now (we can change this), when an azure-cli-acs module does exist, these `acs` commands in azure-cli-vm will not be loaded as the azure-cli-acs module will get loaded instead.

cc @johanste

I am OK to keep it like this for now but something will have to change in the future (possibly moving this to azure-cli-acs) otherwise the acs commands won't be loaded.

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py [hunk 1]
```diff
@@ -159,9 +159,29 @@
...
-    short-summary: Use Docker-based tools to deploy and manage containers
+    short-summary: Manage Azure container services
+"""
+helps['acs create'] = """
+    type: command
+    short-summary: Create a container service with your preferred orchestrator
+"""
+helps['acs delete'] = """
+    type: command
+    short-summary: delete a container service
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]: Does this appear capitalized in the help like `acs create` above?

**Comment by yugangw-msft** [comment]
Got it. `az acs dcos browse` will be contributed by the other team in a brand new component `azure-cli-acs` they planned. I will communicate with them to see whether they can put it in our vm component, but regardless i agree we need to discuss

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py [hunk 2]
```diff
@@ -1239,3 +1239,40 @@ def body(self):
             "image": None
             }
         self.assertEqual(result['storageProfile']['dataDisks'][0], disk2)
+
+class AzureContainerServiceScenarioTest(ResourceGroupVCRTestBase): #pylint: disable=too-many-instance-attributes
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py]: The tests were added but there are no new recordings?

Comment (derekbekoe) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py]: Ah okay GitHub hid the changes so missed it

**Pull request merged**